src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/InputDisplayNameIncorrectAlert/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Text } from 'grommet';
+import React from 'react';
+
+interface InputDisplayNameIncorrectAlert {
+  alertMsg: string;
+}
+
+export const InputDisplayNameIncorrectAlert = ({
+  alertMsg,
+}: InputDisplayNameIncorrectAlert) => {
+  return (
+    <Box pad="1px">
+      <Text color="accent-2" size="11px">
```

Comment:
you should use rem for size and not px here

Reply:
done in [fb4e6bc849131d7bd921ef7f0c208e52364de54a](https://github.com/openfun/marsha/pull/1276/commits/fb4e6bc849131d7bd921ef7f0c208e52364de54a)